### ISSUE TITLE
iCubErzelli01 S/N:032 – [issues with finger joints]

### ISSUE THREAD

**Issue by ddetommaso** [robotology/icub-tech-support#1648]
### Robot Name 🤖

iCubErzelli01 S/N:032

### Request/Failure description

We have some minor issues related to the fingers.
- Displacement. On both hands we have displaces positions of one or two fingers and we would like to align all of them with the same position.
- The index finger of the right hand oscillates repeatedly after a mechanical strain.

### Detailed context

-

### Additional context

https://github.com/robotology/icub-tech-support/assets/3440640/48b00fde-3bb1-4a58-a9d0-ca70dcbadd98

### How does it affect you?

We would need to start data collection in one week and we would like to fix this before starting.

**Comment by AntonioConsilvio** [comment]
Hi @ddetommaso, you can try slightly tightening the following screw:

![Finger](https://github.com/robotology/icub-tech-support/assets/114915464/a99faab8-60b5-4ab4-b9af-cc9638a89b04)

Not too much because you might lock the finger, but slightly might help prevent the finger from making that movement you see in the video.


If the problem persists, we will schedule surgery when possible.

cc @sgiraz

**Comment by sgiraz** [comment]
Hi @ddetommaso,

Is the problem still present? Otherwise, I would proceed to close the issue.

**Comment by ddetommaso** [comment]
Hi @sgiraz,

yes unfortunately the issue is still there.
Can you please schedule an intervention?

**Comment by sgiraz** [comment]
> Hi @sgiraz,
> 
> yes unfortunately the issue is still there. Can you please schedule an intervention?

Yep. We have already scheduled some interventions for other robots in Erzelli for the 2nd week of November. We'll allocate some time for iCubErzelli01 as well.

**Comment by sgiraz** [comment]
/remind 3 November Maintenance activity scheduled for next Monday (cc @Fabrizio69 @GiacomoVassallo13 @AntonioConsilvio)

**Comment by ddetommaso** [comment]
@sgiraz please we need to reschedule the intervention ASAP.

Thank you

**Comment by sgiraz** [comment]
Hi @ddetommaso,

Unfortunately, we can't schedule the intervention before November 28th. Please let me know if you have a preference for a date between November 28th and December 1st.

**Comment by ddetommaso** [comment]
Hi @sgiraz,

let's schedule the intervention for the first day available ensuring to complete the fix in the next days if needed.

**Comment by sgiraz** [comment]
/remind 27 November Maintenance activity scheduled for tomorrow (cc @Fabrizio69 @GiacomoVassallo13 @AntonioConsilvio)

**Comment by AntonioConsilvio** [comment]
Hi! Today I intervened on the robot trying to figure out what the problem on the right index finger was due to. 
After some testing, the problem no longer occurred following a more accurate distal index calibration.

However, the robot had other problems; in fact, the left hand did not calibrate due to a collision between the thumb and index finger that blocked both fingers from reaching the desired position. 
This problem was also solved by adjusting the configuration files.

An additional problem is the adduction of the fingers of the right hand not working. 
This does not cause problems during calibration or operation of the robot, simply the joint does not move.
It does not seem to have any mechanical problems (intact tendon), however when I tested it with the Jogger, the motor sometimes worked and sometimes did not. 
In fact sometimes the motor seemed to stop responding to the +-12V set by the jogger, other times it did.
Being a joint of "minor" importance compared to the others, it was agreed with @ddetommaso to keep it that way for now and see how negatively it affects the tests and demos.

I will prepare the PR on robots-configuration shortly.

cc @sgiraz
